appcues/src/main/java/com/appcues/ui/primitive/OptionSelectPrimitive.kt

```diff
@@ -216,15 +216,15 @@ private fun List<OptionSelectPrimitive.OptionItem>.ComposePicker(
     valueSelectionChanged: (Set<String>) -> Unit,
 ) {
     var expanded by remember { mutableStateOf(false) }
+    val selectedValue = selectedValues.firstOrNull()
     Box(modifier = Modifier.clickable(onClick = { expanded = true })) {
         // 1. render the selected item as the collapsed state
         Row(modifier = modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
             // we should always either have a selected item, or a placeholder, or this will just be a blank
             // box with a dropdown arrow
-            val selectedValue = selectedValues.firstOrNull()
             if (selectedValue != null) {
                 // if we have a selected value, we assume one of our options will match it, and compose it
-                this@ComposePicker.firstOrNull { it.value == selectedValue }?.content?.Compose()
+                this@ComposePicker.firstOrNull { it.value == selectedValue }?.contentView(true)?.Compose()
```

Comment:
This is the picker style. I'm not sure if this is desired, but seems valid... when rendering the "closed" state of the picker with the selected item visible, if an item has a selected state defined - it is used.